### ISSUE TITLE
Can't freeze numexpr 2.4.3

### ISSUE THREAD

**Issue by anthony-tuininga** [marcelotduarte/cx_Freeze#142]
Originally reported by: **Florian Zierer (Bitbucket: [ziererf](https://bitbucket.org/ziererf), GitHub: Unknown)**

----------------------------------------

I used earlier versions of numexpr and there was never a problem freezing it.

I recently upgraded cx_freeze to 4.3.4 and numexpr to 2.4.3 (last known working version was 2.2.2) and get the following error  when running my .exe

```
#!python

  File "tables\__init__.py", line 26, in <module>
  File "numexpr\__init__.py", line 42, in <module>
  File "numexpr\expressions.py", line 45, in <module>
  File "ExtensionLoader_numexpr_interpreter.py", line 24, in <module>
  File "ExtensionLoader_numexpr_interpreter.py", line 16, in __bootstrap__
ImportError: DLL load failed: Das angegebene Modul wurde nicht gefunden.
```

The numexpr.interpreter.pyd file is still included in my build dir as it was with the earlier version.

```
#!python

copying C:\Python27\lib\site-packages\numexpr\interpreter.pyd -> build\exe.win32-2.7\numexpr.interpreter.pyd
```

How can I fix this problem?

If I go back to numexpr-2.1 (I cant build from the sources and only have that binary windows installer) everything works as before.

----------------------------------------
- Bitbucket: https://bitbucket.org/anthony_tuininga/cx_freeze/issue/142

**Comment by anthony-tuininga** [comment]
*Original comment by* **Florian Zierer (Bitbucket: [ziererf](https://bitbucket.org/ziererf), GitHub: Unknown)**:

----------------------------------------

I just found another numexpr installer version 2.4 and this one is also still working.
Should I report that to the numexpr issue tracker?

**Comment by anthony-tuininga** [comment]
*Original comment by* **Florian Zierer (Bitbucket: [ziererf](https://bitbucket.org/ziererf), GitHub: Unknown)**:

----------------------------------------

Seems to be fixed now. I tried with numexpr 2.4.6

**Comment by anthony-tuininga** [comment]
*Original comment by* **Florian Zierer (Bitbucket: [ziererf](https://bitbucket.org/ziererf), GitHub: Unknown)**:

----------------------------------------

Seems fixed with numexpr 2.4.6
